File: Ruby_Course/Basics/return_statement.rb
def cube(num)
  num * num * num
end

puts cube(2)

def cube(num)
  num * num * num
  "last value"
end

 puts cube(2)

 def cube(num)
   return num * num * num
   "last value"
 end

 puts cube(2)

 def cube(num)
   return num * num * num, 70
   "last value"
 end

 puts cube(2)
 puts cube(2)[1]

File: Cpp_Course/Cpp_Course/Files/return_statement.c
#include <iostream>

using namespace std;

double cube(double num){
    double result = num * num * num;
    return result;
}

double cube1(double num){
    return num * num * num;
    cout << "Hello"
}

// return signals C++ to end the function

int main()
{
    double answer = cube(5.0);

    cout << "The result is: " << answer << endl;
    cout << "The result is: " << cube(5.0) << endl;
    cout << "The result is: " << cube1(5.0) << endl;

    return 0;
}
